packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -100,12 +101,15 @@ export class WorkspaceContextUtil {
       try {
         const connection = await this.getConnection();
         this._orgId = connection?.getAuthInfoFields().orgId;
-      } catch (error) {
+      } catch (error: unknown) {
         this._orgId = '';
-        console.log(
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          `There was an problem getting the orgId of the default org: ${error}`
-        );
+        if(error instanceof Error) {
```

Comment:
```suggestion
        if (error instanceof Error) {
```

Reply:
I'm kinda surprised prettier didn't catch this

Reply:
Good eye..yeah I'm not sure how well prettier is integrated in this repo.  The chances are low, but error type validation is just a thing you have to do in Typescript.